cinder/volume/drivers/vmware/vmdk.py

```diff
@@ -2488,17 +2496,31 @@ def revert_to_snapshot(self, context, volume, snapshot):
     def migrate_volume(self, context, volume, host):
         """Migrate a volume to the specified host.
 
-        If the backing is not created, the dest host will create it.
+        If the backing is not created, returns success.
         """
+
+        false_ret = (False, None)
+        if volume['status'] != 'available':
+            return false_ret
```

Comment:
On all those `false_ret`, do we maybe want some logging? Just thinking about customers asking us why something didn't work ...